src/server/mod.rs

```diff
@@ -52,5 +52,9 @@ pub fn is_valid_name(name: &str) -> bool {
             return false;
         }
     }
-    return name != "_localdev"
+    return true;
+}
+
+pub fn is_local_dev(name: &str) -> bool {
```

Comment:
Not sure if we need this function.  `if name == "_localdev"` reads even cleaner than `if is_local_dev(name)`